react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts

```diff
@@ -15,11 +17,10 @@ function getUnfoldedPath(object: Feature) {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const a: any[] = [];
     delta.forEach((d) => {
-        const prev = a.at(-1) || 0;
+        const prev = a.at(-1) || first_x;
```

Comment:
World coordinates don't make sense in this projection. Instead X should mean distance from the beginning of the fence. We should change the axis origin in this viewport 0.

Reply:
Changed back to 0